packages/http/src/webauthn.ts

```diff
@@ -131,7 +136,20 @@ export async function getWebAuthnAssertion(
 export async function getWebAuthnAttestation(
   options: TurnkeyCredentialCreationOptions
 ): Promise<TAttestation> {
+  const webAuthnSupported = hasWebAuthnSupport();
+
+  if (!webAuthnSupported) {
+    throw new Error("webauthn is not supported by this browser");
+  }
+
   const res = await webauthnCredentialCreate(options);
 
   return toInternalAttestation(res.toJSON());
 }
+
+// `hasWebAuthnSupport` checks for barebones webauthn support.
+// For additional details and granular settings, see:
+// https://web.dev/articles/passkey-form-autofill#feature-detection, https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential
+function hasWebAuthnSupport(): boolean {
```

Comment:
a little more verbose, but choosing to have this separate function for now in case we want to modify this shared logic